payjoin-io/src/lib.rs

```diff
@@ -0,0 +1,2 @@
+pub mod error;
+pub mod ohttp_keys;
```

Comment:
Since the very first iteration does only one thing it makes more sense to me to put all of the functionality in lib.rs. We probably don't have modules to separate yet.

Reply:
I did this after I added the `Error` and `ProxyServer` and  `lib.rs` didnt feel like the right place for them. with that said, if following the comment above we make `payjoin-io` depend on `payjoin` for the error object and other types, that could trim some of the code I think then it would make sense to move everything to `lib.rs`